projects/perp88/index.js

```diff
@@ -10,6 +11,48 @@ const tokens = {
 }
 const POOL_DIAMOND_CONTRACT = '0xE7D96684A56e60ffBAAe0fC0683879da48daB383';
 
+async function blastTvl(_, _1, _2, { api }) {
+  const vaultStorageAddress = '0x97e94BdA44a2Df784Ab6535aaE2D62EFC6D2e303'
+  const hlpUnderlyings = [
```

Comment:
added code to price the yield bearing tokens here: https://github.com/DefiLlama/defillama-server/commit/ea66a702a064c0cd8c39aa2120fcde8686831017#diff-31b49e7a59545ee10ded3ab0cfc3429ad5a5365b3495a7a2c0fbb0eb804d3fa8R39 so this adaper can be simplified